src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,16 +1,21 @@
 import React from 'react';
 import { Movie } from '../../types/Movie';
-import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
 
 type Props = {
-  movies: Movie[];
+  movies: Movie[]
 };
 
-export const MoviesList: React.FC<Props> = ({ movies }) => (
+export const MoviesList: React.FC <Props> = ({ movies }) => (
   <div className="movies">
+
     {movies.map(movie => (
-      <MovieCard key={movie.imdbId} movie={movie} />
+      <div key={movie.imdbId} className="card" data-cy="Movie">
+
+        <MovieCard movie={movie} />
+
+      </div>
```

Comment:
```suggestion
      <MovieCard movie={movie} key={} />
```
The `.card` is a part of the MovieCard component. The key attribute can be added to the MovieCard too.